docs/program/extend/modular-resources/examples/add-tflite-module.md

```diff
@@ -0,0 +1,325 @@
+- [`example_audio_classification_client.cpp`](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/src/viam/examples/mlmodel/example_audio_classification_client.cpp) - an example inference client which generates audio samples and invokes the `example_mlmodelservice_tflite` module to classify those samples using the [`yamnet/classification` TensorFlow Lite model](https://tfhub.dev/google/lite-model/yamnet/classification/tflite/1).
+
+This tutorial walks you through everything necessary to start using these example files with your robot, including building the C++ SDK, configuring your robot and installing `viam-server`, and generating results with the example inference client program.
+
+## Build the C++ SDK
+
+To build the Viam C++ SDK, you will need a macOS or Linux computer.
+Follow the instructions below for your platform:
+
+{{< tabs >}}
+{{% tab name="macOS" %}}
+
+Follow the [Viam C++ SDK build instructions](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/BUILDING.md) to build the SDK on your macOS computer using the `brew` package manager.
+
+While your specific build steps may differ slightly, your installation should generally resemble the following:
+
+1. Install all listed dependencies to support the Viam C++ SDK:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   brew install abseil cmake boost grpc protobuf xtensor pkg-config ninja buf
+   ```
+
+1. Create a new <file>example_workspace</file> directory for this tutorial and clone the Viam C++ SDK:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   mkdir -p ~/example_workspace
+   cd ~/example_workspace
+   git clone git@github.com:viamrobotics/viam-cpp-sdk.git
+   ```
+
+1. Create an <file>opt</file> directory to install the build artifacts to:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   mkdir -p ~/example_workspace/opt
+   ```
+
+1. Create a <file>build</file> directory to house the build:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   cd viam-cpp-sdk/
+   mkdir build
+   cd build
+   ```
+
+1. Create an environment variable `PKG_CONFIG_PATH` which points to the version of `openssl` installed on your system:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   export PKG_CONFIG_PATH="`brew --prefix`/opt/openssl/lib/pkgconfig"
+   ```
+
+1. Build the C++ SDK by running the following commands:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   cmake .. -DVIAMCPPSDK_BUILD_TFLITE_EXAMPLE_MODULE=ON -DVIAMCPPSDK_USE_DYNAMIC_PROTOS=ON -DCMAKE_INSTALL_PREFIX=~/example_workspace/opt -G Ninja
+   ninja all
+   ninja install
+   ```
+
+   This tutorial passes three flags to the build process to configure the build:
```

Comment:
On first read I wondered, does the tutorial pass them? Do the steps above pass them? Do I as a user need to be sure to pass them at some point in the future?
```suggestion
   The command above passes three flags to the build process to configure the build:
```